### ISSUE TITLE
No alt text for SUL logos

### ISSUE THREAD

**Issue by jvine** [sul-dlss/SearchWorks#971]
At top of page:
```
<a href="http://library.stanford.edu">
      <span class="sr-only">Stanford University Libraries</span>
      <img alt="" class="hidden-xs" data-hidpi-src="/assets/sul-logo@2x-a5dfe34dd21c1c638a102ecb4b8f5d27.png" height="25" src="/assets/sul-logo-ef484810477430a2f28660bd6d54ba0f.png">
      <img alt="" class="visible-xs" data-hidpi-src="/assets/sul-logo-small@2x-a8d58f36750de2a8fc7c1d7d874bba76.png" height="25" src="/assets/sul-logo-small-e766d2e0df676a1c82ff09ab7d1073a0.png">
</a>
```
The .sr-only text should be the alt attribute, so:
```
<a href="http://library.stanford.edu">
      <img alt="Stanford University Libraries" class="hidden-xs" data-hidpi-src="/assets/sul-logo@2x-a5dfe34dd21c1c638a102ecb4b8f5d27.png" height="25" src="/assets/sul-logo-ef484810477430a2f28660bd6d54ba0f.png">
      <img alt="Stanford University Libraries" class="visible-xs" data-hidpi-src="/assets/sul-logo-small@2x-a8d58f36750de2a8fc7c1d7d874bba76.png" height="25" src="/assets/sul-logo-small-e766d2e0df676a1c82ff09ab7d1073a0.png">
</a>
```

And in the footer:
```
<img alt="" data-hidpi-src="/assets/SUL-logo-stacked@2x-d62749d10a6abb9e854221a499bc68f8.png" height="45" src="/assets/SUL-logo-stacked-112b90426f8dca04d5322306ca82a0d3.png">
```
We should probably also link the logo to be parallel with the SU footer, so:

```
<a href="http://library.stanford.edu">
<img alt="Stanford University Libraries" data-hidpi-src="/assets/SUL-logo-stacked@2x-d62749d10a6abb9e854221a499bc68f8.png" height="45" src="/assets/SUL-logo-stacked-112b90426f8dca04d5322306ca82a0d3.png">
</a>
```

(Yes, it's redundant.)
